include/swift/AST/SILOptions.h

```diff
@@ -56,6 +56,11 @@ class SILOptions {
   /// Controls how to perform SIL linking.
   LinkingMode LinkMode = LinkNormal;
 
+  /// Controls whether to pull in SIL from partial modules during the
+  /// merge modules step. Could perhaps be merged with the link mode
+  /// above but the interactions between all the flags are tricky.
```

Comment:
I think it's worth trying to merge them, in the spirit of "not leaving work around to clean up later". The settings are already not orthogonal.